nservicebus/upgrades/5to6-moving-away-from-ibus.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Moving away from `IBus` in NServiceBus Version 6
+summary: Describes how to send messages without the IBus
+---
+
+Starting with Versions 6 and above, the `IBus` interface has been deprecated.
+
+The `IBus` interface was one of the key interfaces when using previous versions of NServiceBus. It provided access to many of the important operations like sending messages, subscribing to events, and manipulating headers. The `IBus` interface was available through dependency injection in message handlers and any custom components that were registered in the container. 
+
+Some scenarios involving `IBus` include endpoint creation, sending messages during endpoint startup, sending messages from within message handlers, using the injected `IBus` in custom components also registered in the container. 
+
+## During Endpoint Creation
+
+In previous versions of NServiceBus to start a new instance of an endpoint, either the `Bus` static class or the `IStartableBus` interface was used. In Version 6 these two concepts have been replaced. More information can be found in the [hosting](/nservicebus/hosting/) section of the documentation.
```

Comment:
@andreasohlund - I didn't add this, I am just moving this from one file to another. What needs to go here?